### PR TITLE
[risk=no][DB-502]source-tree-bugs

### PR DESCRIPTION
**Pull request by mhigbyflowers**
tree works much better. only one sibling can be opened at a time but it works better
also got rid of the spacing bug.

### CODE REVIEW

File: public-ui/src/app/components/recursive-tree/recursive-tree.component.ts [hunk 1]
```diff
@@ -40,7 +40,7 @@ export class RecursiveTreeComponent implements OnChanges, OnDestroy {
       const item = nodes[i];
       if (item === node && i === index) {
         this.childIndex = i;
-        this.subOpened = !this.subOpened;
+        this.subOpened = true;
```
Comment (SrushtiGangireddy) [inline public-ui/src/app/components/recursive-tree/recursive-tree.component.ts:43]: when would this change to false on closing ?

Comment (mhigbyflowers) [reply public-ui/src/app/components/recursive-tree/recursive-tree.component.ts:43]: the component starts as false.
and when another node is clicked it sends the 'opened' var as false, which is used to render/unrender the children'

**Review by SrushtiGangireddy — APPROVED** [review]

File: public-ui/src/app/components/recursive-tree/recursive-tree.component.ts (hunk 1 above)
Comment (mhigbyflowers) [reply public-ui/src/app/components/recursive-tree/recursive-tree.component.ts:43]: kinda hard to explain because of recursion

Comment (SrushtiGangireddy) [reply public-ui/src/app/components/recursive-tree/recursive-tree.component.ts:43]: ohh yeah gotcha. Was circling over there. thanks.

**Pull request merged**